src/products/components/ProductVariants/ProductVariants.tsx

```diff
@@ -360,6 +374,16 @@ export const ProductVariants: React.FC<ProductVariantsProps> = props => {
                       )
                     )}
                   </TableCell>
+                  <TableCell
+                    className={classes.colActions}
+                    data-test="actions"
+                    onClick={e => e.stopPropagation()}
+                  >
+                    <ProductVariantSetDefault
```

Comment:
it's nothing important, but when there is no children I think it's better:                   
```
  <ProductVariantSetDefault
        variant={variant}
        productId={productId}
      />
```

Reply:
That's strange, because we have git hook on commit that runs prettier which should format this